README.md

```diff
@@ -1,87 +1,100 @@
 
-Atomic App itself is packaged as a container. End-users typically do not install the software from source. Instead use the `atomicapp` container as the `FROM` line in a Dockerfile and package your application on top. For example:
+  - __A "packaged installer" for all container-based environments and applications.__ Replace all those bash and Ansible scripts with one container-based deployment tool.
 
-```
-FROM projectatomic/atomicapp
-
-MAINTAINER Your Name <you@example.com>
-
-ADD /Nulecule /Dockerfile README.md /application-entity/
-ADD /artifacts /application-entity/artifacts
-```
+  - __Target multiple providers:__ Specify the provider you want the Atomic App to run on. It supports Kubernetes, OpenShift, Mesos+Marathon and Docker.
 
-For more information see the [Nulecule getting started guide](https://github.com/projectatomic/nulecule/blob/master/docs/getting-started.md).
+  - __Inherit already packaged containers:__ Create composite applications by referencing other Nulecule-compliant applications. For example, plugging in an alternative well-orchestrated database in another referenced container image.
 
-## Developers
+  - __Fetch and run entire environments:__ Use `atomicapp fetch` and `atomicapp run` to run pre-packaged Nuleculized containers.
 
-First of all, clone the github repository: `git clone https://github.com/projectatomic/atomicapp`.
-
-### Installing Atomic App locally
-Simply run
-
-```
+## Installing Atomic App
+From Linux:
+```sh
+git clone https://github.com/projectatomic/atomicapp && cd atomicapp
 make install
 ```
 
-If you want to do some changes to the code, I suggest to do:
+_or_ 
 
-```
-cd atomicapp
-export PYTHONPATH=`pwd`:$PYTHONPATH
-alias atomicapp="python `pwd`/atomicapp/cli/main.py"
+Download a pre-signed .tar.gz from [download.projectatomic.io](https://download.projectatomic.io) / [GitHub](https://github.com/projectatomic/atomicapp/releases):
+```sh
+export RELEASE=0.4.2
+wget https://github.com/projectatomic/atomicapp/releases/download/$RELEASE/atomicapp-$RELEASE.tar.gz
+tar -xvf atomicapp-$RELEASE.tar.gz && cd atomicapp-$RELEASE.tar.gz
+make install
 ```
 
-### Building for containerized execution
-```
-docker build -t [TAG] .
-```
+## Documentation
 
-Use 'docker build' to package up the application and tag the resulting image.
+1. [Quick start](docs/quick_start.md)
+2. [Getting started](docs/start_guide.md)
+3. [Providers](docs/providers.md)
+  1. [Docker](docs/providers/docker/overview.md)
+  2. [Kubernetes](docs/providers/kubernetes/overview.md)
+  3. [OpenShift](docs/providers/openshift/overview.md)
+4. [CLI](docs/cli.md)
+5. [Nulecule file](docs/nulecule.md)
+6. [Atomic App lifecycle](docs/atomicapp_lifecycle.md)
+7. [File handling](docs/file_handling.md)
+8. [Specification coverage](docs/spec_coverage.md)
+9. [Contributing](CONTRIBUTING.md)
+10. [Dependencies](docs/requirements.md)
```

Comment:
Hmm. I wanted to follow something along the lines of how docker and kubernetes do their documentation. i quite like that there is an index that shows direct links to the files I'd like to view.

having it near / at the top of the readme.md would be great.

You're right about the headers (some of them are duplicate headers in the readme that link to the same doc)